src/frontend/components/projects/project-card.svelte

```diff
@@ -1,147 +1,57 @@
+    import VersionsDropdown from "./versions-dropdown.svelte";
+    import { Pencil } from "phosphor-svelte";
 
     export let project : Project;
 
-    let expanded = false;
-    let versionsList = project.listVersions();
-    let latestVersion = versionsList.sort((versionA, versionB) => versionB.localeCompare(versionA))[0]
+    const editingName = writable(false);
+    const projectName = writable(project.projectName);
 
-    let configs : Writable<SystemConfigurationStore[]> = writable([]);
+    let latestVersion = project.versions.sort((a,b) =>
+      b.createdAt.localeCompare(a.createdAt) || // Sort by date
+        b.version.localeCompare(a.version), // If dates are equal, sort by version
+    )[0];
 
-    systemConfigurationsStore.items.subscribe(items => {
-        if(!expanded) return;
-        configs.set(items
-            .filter(config => get(config.projectId) === project.identifier) // Filters current project versions
-            .sort((versionA, versionB) => 
-                (get(versionB.updatedAt).getTime() - get(versionA.updatedAt).getTime()) || // Sort By last update
-                (get(versionB.version).localeCompare(get(versionA.version))) // Otherwise, sort by version
-            ))
-        }
-    )
-
-    // configs.subscribe(_configs => {
-    //     versionsList = project.listVersions();
-    //     latestVersion = versionsList.sort((versionA, versionB) => versionB.localeCompare(versionA))[0]
-    // })
-
-    function editVersion (version : SystemConfigurationStore) : void {
-        console.log("Should go to version, for now just logs:", version);
-    }
-
-    function duplicateVersion (version : SystemConfigurationStore) : void {
-        FileSystemController.duplicateConfiguration(version.toEntity(), project);
-        //TODO update after duplication
-    }
-    
-    function deleteVersion (version : SystemConfigurationStore) : void {
-        // TODO Alert before deletion
-        FileSystemController.removeConfiguration(project, version.toEntity());
-    }
 
     function getRelevantUpdateInfo () : string {
-        const values = {
-            year: 31556952000,
-            month: 2629746000,
-            day: 86400000,
-            hour: 3600000,
-            minute: 60000,
-        }
+      const values = {
+        year: 31556952000,
+        month: 2629746000,
+        day: 86400000,
+        hour: 3600000,
+        minute: 60000,
+      };
 
-        const value = new Date().valueOf()-project.updatedAt.valueOf();
-        for(const key in values) {
-            let div =  value / values[key];
-            if( div > 1) return `${Math.floor(div)} ${key}${div >= 2 ? "s" : ""} ago` 
-        }
-        return "moments ago";
+      const value = new Date().valueOf()-project.updatedAt.valueOf();
+      for(const key in values) {
+        let div =  value / values[key];
+        if(div > 1) return `${Math.floor(div)} ${key}${div >= 2 ? "s" : ""} ago`;
+      }
+      return "moments ago";
```

Comment:
I commented this before, but it is not here somehow.

Although I appreciate the effort, let's not maintain this implementation ourselves, as the project already has `date-fns` as a dependency, and it covers this use case.